### PR TITLE
Add PrivateStorage payment server with setup workflow disabled

### PR DESCRIPTION
**Pull request by btlogy**
Same purpose as #2, but trying to avoid the extra commits...

### CODE REVIEW

**Comment by btlogy** [comment]
@hacklschorsch what about reviewing and merging this one and closing #2 ?

**Comment by btlogy** [comment]
Though, we would miss the problem checks for #1 have raised:

* #2

Right: you did fix that in https://github.com/LeastAuthority/end-users-upptime/pull/2/commits/9124e35d89aed71f73643ccd5ca16bca96711735

We can you cherry pick...

**Comment by btlogy** [comment]
I think this could work (and not pollute the master branch with those auto-commits).
@hacklschorsch do you think we could monitor the payment API with an HTTP request?
Or do you think we better stop at the TCP level?

**Comment by btlogy** [comment]
I'm not sure how we should handle this kind of repo. (w/ or w/o PR), but I've been testing changes [there](https://github.com/la-test/sbx1-upptime) to avoid polluting the "states" here.
And this one works: https://la-test.github.io/sbx1-upptime/history/private-storage-payment-api

**Comment by hacklschorsch** [comment]
> @hacklschorsch do you think we could monitor the payment API with an HTTP request?

AFAIK the payment API does not have a public HTTP endpoint that is no-change or idempotent, returns non-error and is public.  The monitoring endpoint would make a nice "are you up at all?" checking endpoint, but is only accessible from within the monitoring VPN.

Really I think the tcp only thing is fine for a first consumer-facing status page.

File: .upptimerc.yml [hunk 1]
```diff
@@ -31,10 +31,9 @@ sites:
     url: https://private.storage/
   - name: Private Storage - payment API
     icon: https://raw.githubusercontent.com/LeastAuthority/end-users-upptime/master/assets/private-storage-icon.svg
-    url: https://payments.private.storage/v1/redeem
-    method: POST
```
Comment (hacklschorsch) [inline .upptimerc.yml]: Please don't do this. A "post" is always meant to create or modify state on the server.  This is not a well-suited liveliness-check endpoint (and screws over our statistics).

**Comment by hacklschorsch** [comment]
The best would probably be a end-to-end check - the second best I could imagine would be a dedicated HTTP "ping" endpoint in the app server (so the reverse proxy is passed and the actual payment server is exercised at least).
At least currently that effort isn't really justified though IMHO

**Pull request merged**